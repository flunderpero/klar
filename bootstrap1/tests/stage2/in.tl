let s = "Hi there"
let needle = "the"
let not_needle = "not"
if "the" in s then
    print("OK: 'the' has been found")
end
if "not" in s then
    print("ERR: 'not' has been found")
end
if needle in s then
    print("OK: 'the' (from var) has been found")
end
if not_needle in s then
    print("ERR: 'not' (from var) has been found")
end
