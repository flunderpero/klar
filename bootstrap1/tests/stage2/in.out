OK: 'the' has been found
OK: 'the' (from var) has been found
